pdd.json

```diff
@@ -0,0 +1,22 @@
+{
+    "version": "1.5",
+    "description": "Date/time difference calculator and countdown timer. The name 'pdd' stands for Python Date Diff.",
+    "homepage": "https://github.com/jarun/pdd",
+    "license": "GPL-3.0-or-later",
+    "notes": [
+        "pdd requires dateutil to run properly",
+        "You can install it via pip: pip install python-dateutil"
+    ],
+    "suggest": {
+        "Python": "python"
+    },
+    "url": "https://github.com/jarun/pdd/raw/v1.5/pdd#/pdd.py",
+    "hash": "298d0b95f37e3cada7d662980e7bc04e9d76a1039df8b6022e1bd6e2f4cb289e",
+    "pre_install": "Set-Content \"$dir\\pdd.bat\" \"python `\"%~dp0pdd.py`\" %*\" -Encoding ascii",
```

Comment:
We don't need to do this. `.py` files are handled by the shimming function. https://github.com/ScoopInstaller/Scoop/blob/b96abcfda9506e820a0d6d6b42befa1bf4971034/lib/core.ps1#L721

Reply:
Ok. Thanks for the info.